resources/definitions/fdmprinter.def.json

```diff
@@ -3379,6 +3379,30 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "support_skip_some_zags":
+                {
+                    "label": "Skip Some ZigZags Connections",
+                    "description": "Skip some ZigZags connections to make the support structure easier to break.",
```

Comment:
'ZigZags' -> 'support line'

'easier to break' -> 'easier to break away'

PS: note that the common term for this type of support is 'break-away support' in contrast to soluble support.